libraries/scrcpy/src/adb/client.ts

```diff
@@ -178,8 +203,8 @@ export class AdbScrcpyClient {
         adb: Adb,
         path: string,
         version: string,
-        options: AdbScrcpyOptions<ScrcpyOptionsInit1_16>
-    ): Promise<string[]> {
+        options: AdbScrcpyOptions<object>
+    ): Promise<ScrcpyEncoder[]> {
         Object.assign(options.value, {
```

Comment:
Add a hook in ScrcpyOptions to set list encoder option (`encoderName: _` vs `listEncoder: true`)